src/itest/resources/bpmn/testForm.json

```
@@ -84,30 +84,29 @@
           ]
         },
         {
-          "legend": "Reference table",
-          "type": "referenceTableFieldset",
-          "key": "RT_ReferenceTable",
+          "legend": "Available Documents",
+          "type": "documentsFieldset",
+          "key": "ZAC_Documents",
```

```suggestion
              "key": "ZAAK_Documents",
```
Maybe better? This is not across the whole of ZAC but just for the Zaak.